### PR TITLE
Rework toolbelt to match new apps implementation

### PR DESCRIPTION
**Pull request by tamorim**
Depends on https://github.com/vtex/apps-client-node/pull/3

### CODE REVIEW

File: src/modules/auth.js [hunk 1]
```diff
@@ -70,37 +113,79 @@ function promptUsePrevious () {
...
+}
+
+function workspaceExists (account, workspace, token) {
+  return new VBaseClient({
+    authToken: token,
+    userAgent: userAgent,
+  }).get(account, workspace)
+  .then(() => true)
+  .catch(res =>
+    res.error.Code === 'NotFound'
```
Comment (firstdoit) [inline src/modules/auth.js]: What if `res.error == undefined`? :boom:

File: src/modules/auth.js [hunk 2]
```diff
@@ -70,37 +113,79 @@ function promptUsePrevious () {
...
+        ])
       })
-      .tap(([account, login, token]) => {
-        credentials = {account, login, token}
-        log.debug('Creating sandbox workspace...')
+      .then(([account, login, token, workspace]) => {
+        log.debug('Checking if workspace exists...')
+        return workspaceExists(account, workspace, token)
+        .then(exists => {
+          if (exists) return workspace
```
Comment (firstdoit) [inline src/modules/auth.js]: Ooohh didn't our linter catch this? `if` without curly braces! :boom:

File: src/modules/workspace.js [hunk 3]
```diff
@@ -67,11 +69,15 @@ export default {
...
-        .then(() => saveCurrentWorkspace(name))
+        .then(() => saveWorkspace(name))
+        .then(() => log.info(`You're now using the workspace ${name}!`))
         .catch(res => {
-          return res.statusCode === 404
-          ? log.info(`Workspace ${name} not found`)
-          : Promise.reject(res)
+          if (res.error.Code === 'NotFound') {
+            log.info(`Workspace ${name} doesn't exist`)
+            log.info(`You can use ${chalk.bold(`vtex workspace create ${name}`)} to create it!`)
```
Comment (firstdoit) [inline src/modules/workspace.js]: Nice touch ;)

**Comment by firstdoit** [comment]
Some small issues, then LGTM 👍

**Comment by tamorim** [comment]
@firstdoit Done!

**Comment by firstdoit** [comment]
💥

**Pull request merged**